app/controllers/concerns/paginatable.rb

```diff
@@ -16,6 +16,17 @@ def page_from_params(params)
         page = {}
       end
 
+      # All cursors will need to be decoded from the param
+      if page[:cursor].present?
+        begin
+          # When we decode and split, we'll always end up with an array
+          page[:cursor] = Base64.strict_decode64(page[:cursor]).split(",")
+        rescue ArgumentError
+          # If we fail to decode we'll just default back to an empty cursor
+          page[:cursor] = []
```

Comment:
Note we don't want to throw an error back here because cursor=1 is allowed in our API and we need to kick start cursors sometimes.

Reply:
Looks good.